### PR TITLE
Write all PVs to LVM devices file

### PR DESCRIPTION
**Pull request by VladimirSlavik**
Resolves: [rhbz#2040302](https://bugzilla.redhat.com/show_bug.cgi?id=2040302)

### CODE REVIEW

File: pyanaconda/modules/storage/installation.py [hunk 1]
```diff
@@ -337,12 +337,15 @@ def _write_lvm_devices_file(self, storage, sysroot):
 
         Adds all present PVs according to https://bugzilla.redhat.com/show_bug.cgi?id=2011329#c9
 
+        /etc/lvm/devices/system.devices
```
Comment (poncovka) [inline pyanaconda/modules/storage/installation.py:340]: This piece of information kind of comes out of nowhere.

File: pyanaconda/modules/storage/installation.py [hunk 2]
```diff
@@ -337,12 +337,15 @@ def _write_lvm_devices_file(self, storage, sysroot):
...
+
         :param Blivet storage: instance of Blivet or a subclass
         :param str sysroot: path to the target OS installation
         """
-        for device in storage.devices:
-            if device.format and device.format.type == "lvmpv" and HAVE_LVMDEVICES:
-                device.format.lvmdevices_add()
+        for devices in (storage.devices, storage.devicetree._hidden):
```
Comment (poncovka) [inline pyanaconda/modules/storage/installation.py:345]: You can use `itertools.chain` to iterate over both lists of devices.

Comment (VladimirSlavik) [reply pyanaconda/modules/storage/installation.py:345]: Sounds good! I believed you would want it the other way, less imports :-)

Comment (poncovka) [reply pyanaconda/modules/storage/installation.py:345]: Imports from standard libraries are never an issue :-)

**Comment by VladimirSlavik** [comment]
Update: changed to use `itertools.chain`.

**Review by poncovka — APPROVED** [review]
It looks great. Thank you!

**Pull request merged**